### DISCUSSION TITLE
Ability to see scenario failures

### DISCUSSION THREAD

**Discussion by fkhamdamov** [yaalalabs/fixyl#16] [Ideas]
Would be good to see why scenario failed instead just having failed status. Like report with what didn't match between input and expected output.

**Comment by DhananjayaR** [comment]
Thanks for the suggestion. Will do the necessary changes in a future release and add an update here.

**Reply by DhananjayaR** [reply]
Issue fixed in https://github.com/yaalalabs/fixyl/releases/tag/v1.0.8
